code-challenges.test.js

```diff
@@ -11,41 +11,113 @@
+//       expect(multipliedByThree(numsArray1)).toEqual([18, 21, 24, 27, 30])
+//       expect(multipliedByThree(numsArray2)).toEqual([72, 81, 90, 99, 108])
+//     })
+//   })
+  
+//   // b) Create the function that makes the test pass.
+  
+//   // create a function that takes in an array of numbers
+//   const multipliedByThree = (arr) => {
+//     // iterate using .map
+//     return arr.map((value) => {
+//         // return product in the new array created by .mapx
+//         //return an array with all the numbers multiplied by 3
+//         return value * 3
+//     })
+//   }
 
 // Pseudo code:
 
-// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
-// a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
 
-// b) Create the function that makes the test pass.
+// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
-// Pseudo code:
 
+// Function to check if a number inside an object is evenly divisible by three
+// const checkDivisibileByThree = (divisible) => {
+//   const { number } = divisible
+//   if (number % 3 === 0) {
+//     return `${number} is divisible by 3`
+//   } else {
+//     return `${number} is not divisible by 3`
+//   }
+// }
+
+// // jest test using describe
+// describe('Check Divisibile By Three', () => {
+//   const object1 = { number: 15 }
+//   // Expected output: "15 is divisible by three"
+//   const object2 = { number: 0 }
+//   // Expected output: "0 is divisible by three"
+//   const object3 = { number: -7 }
+//   // Expected output: "-7 is not divisible by three"
+// // jest test using it
+//   it('should return "15 is divisible by three"', () => {
+//     //jest test using expect with
+//       expect(checkDivisibileByThree(object1)).toBe("15 is divisible by 3")
+//   })
+
+//   it('should return "0 is divisible by three"', () => {
+//       expect(checkDivisibileByThree(object2)).toBe("0 is divisible by 3")
+//   })
+
+//   it('should return "-7 is not divisible by three"', () => {
+//       expect(checkDivisibileByThree(object3)).toBe("-7 is not divisible by 3")
+//   })
+// })
+
+// // Pseudo code:
+// // a) Create a test with expect statements for each of the variables provided.
+// // b) Create the function that makes the test pass.
+// // c) Create the anatomy of a Jest Test with describe statement to set a parameter
+// // d) Create a -it- Jest Test to inform us what the describe statement should be doing
+// e) Create an -expect- Jest Test to have a argument to invoke
```

Comment:
Where is your pseudo code for the actual function? You wrote "function to check if a number inside an object is evenly divisible by three" but that doesn't explain how to code is doing what it's doing at all. This pseudo code on lines 85-90 is describing your jest test, which you don't need pseudo code for at all.